index.html

```diff
@@ -0,0 +1,176 @@
+                        <li>
+                            <a href="#" class="active-tab">Home</a>
+                        </li>
+                        <li>
+                            <a href="about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="index.html#main-program">Program</a>
+                        </li>
+                        <li>
+                            <a href="https://apply.microverse.org/" target="_blank" rel="noopener">Join</a>
+                        </li>
+                    </ul>
+            </nav>
+        </header>
+        <section class="introduction">
+            <h1 class="introduction__header">
+                Microverse Full-Stack web development program
+            </h1>
+            <div class="introduction__description-container">
+                <p class="introduction__description">
+                   A program to get you started on Full-Stack web development with Technical and Soft skills required, This  program create positive changes on their alumni, You can join the course wherever you are.
+                </p>
+            </div>
+            <h2 class="event-date">
+                October class: 2021.11.18 (MON) 
+            </h2>
+            
+        </section>
+        
+        <section class="main-program" id="main-program">
+            <h3>
+                Main Program
+            </h3>
+            <div class="main-program__container">
+                <div class="card">
+                    <img src="images/personnel.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Personnel</span>
+                    <p class="card-description">Checkout Microverse staff members </p>
+                </div>
+
+                <div class="card">
+                    <img src="images/course-icon.png" alt="monitor with a book inside it">
+                    <span class="card-title">Curriculum</span>
+                    <p class="card-description">Checkout Microverse program curriculum </p>
+                </div>
+
+                <div class="card">
+                    <img src="images/forum.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Forum</span>
+                    <p class="card-description">Have the time sharing your thoughts and opinion</p>
+                </div>
+
+                <div class="card">
+                    <img src="images/alumni.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Microverse Alumni</span>
+                    <p class="card-description">See Microverse first graduated students opinion on their experience</p>
+                </div>
+                <a href="https://apply.microverse.org/" target="_blank" rel="noopener" class="join-btn" id="join-btn">
+                Apply now
+                </a>
+            </div>
+        </section>
```

Comment:
- [ ] Please make use of the `article` semantic HTML tag wherever necessary instead of divs in order to comply with the project requirements about HTML best practices.